lib/ancestry/has_ancestry.rb

```diff
@@ -30,6 +30,13 @@ def has_ancestry options = {}
       validates_format_of self.ancestry_column, :with => derive_ancestry_pattern(options[:primary_key_format]), :allow_nil => true
       extend Ancestry::MaterializedPath
 
+      @@default_update_strategy = :ruby
```

Comment:
I think we want to turn this on for all tests in `environment.rb`


When/if we have optimizers for all databases we may decide to introduce a ruby test, but in the meantime, mysql and sqlite will test these values

Reply:
sorry I was not clear here